vpnmanager/vpnservicemanager/src/main/java/com/kape/vpnservicemanager/data/externals/Service.kt

```diff
@@ -82,6 +83,7 @@ internal class Service :
         protocol.stopConnection(disconnectReason).getOrElse {
             return Result.failure(it)
         }
+        gateway = null
```

Comment:
this is not needed once we move it to `cache`.

Reply:
addressed.